src/views/startup_submission.hbs

```diff
@@ -5,22 +5,22 @@
 <form class="" action="/registerStartup" method="post">
 
   <fieldset>
-    <label for="startup-name">Startup Name: </label>
+    <label for="startup-name">Startup Name </label>
```

Comment:
Im not sure that we need the word Startup before each property.
what do you think about just leaving it as Name, Description, Stage..?